suite-common/token-definitions/scripts/utils/sign.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-console */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as jws from 'jws';
+
+import { JWS_SIGN_ALGORITHM } from '../../src/constants';
+import { TokenStructure } from '../../src/types';
+
+// There must be no extra spaces at the beginning of the line.
+const devPrivateKey = `-----BEGIN EC PRIVATE KEY-----
+MHQCAQEEINi7lfZE3Y5U9srS58A+AN7Ul7HeBXsHEfzVzijColOkoAcGBSuBBAAKoUQDQgAEbSUHJlr17+NywPS/w+xMkp3dSD8eWXSuAfFKwonZPe5fL63kISipJC+eJP7Mad0WxgyJoiMsZCV6BZPK2jIFdg==
+-----END EC PRIVATE KEY-----`;
+
+const getPrivateKey = () => {
+    // Only CI jobs flagged with "codesign", sign message system config by production private key. All other branches use development key.
+    // The isCodesignBuild() util cannot be used here because the lib is not built at this point. Building libs would make the release script slower.
+    if (process.env.IS_CODESIGN_BUILD !== 'true') {
+        console.log('Signing config using develop private key!');
+
+        return devPrivateKey;
+    }
+
+    console.log('Signing config using production private key!');
+
+    const privateKey = process.env.JWS_PRIVATE_KEY_ENV; // available on GitHub
```

Comment:
I don't want to look for problems, just thinking out loud - is it a good idea to use the same private key for the message system and different token definitions? @vdovhanych ?

Reply:
I don't really know what the implications of leaking the key for this or the msg system configuration are. Can this be used maliciously in the same way as a message system? But then again would that help us to use different key for that?